internal/pkg/selinux/selinux.go

```diff
@@ -45,15 +46,26 @@ func IsEnforcing() bool {
 	return val != nil && *val == "1"
 }
 
-// SetLabel sets label for file or directory, following symlinks
-// It does not perform the operation in case SELinux is disabled or provided label is empty.
+// SetLabel sets label for file, directory or symlink (not following symlinks)
+// It does not perform the operation in case SELinux is disabled, provided label is empty or already set.
 func SetLabel(filename string, label string) error {
 	if label == "" {
 		return nil
 	}
 
 	if IsEnabled() {
```

Comment:
Further optimization: add a caching variable in `IsEnabled`. Maybe I should add it as another change and split out this change (it's not directly required for mounts).